controller/api/destination/server_test.go

```diff
@@ -97,7 +97,7 @@ spec:
 	if err != nil {
 		t.Fatalf("NewFakeAPI returned an error: %s", err)
 	}
-	log := logging.WithField("test", t.Name)
+	log := logging.WithField("test", t.Name())
```

Comment:
This was causing an error to be added in the log output: `logrus_error="can not add field \"test\"`

Reply:
Looks like we do this a lot:

```
$ ag 'WithField\("test", t.Name\)'
controller/tap/handlers_test.go
57:				log:    logrus.WithField("test", t.Name),

controller/api/destination/profile_translator_test.go
436:			log:    logging.WithField("test", t.Name),
456:			log:    logging.WithField("test", t.Name),
476:			log:    logging.WithField("test", t.Name),
492:			log:    logging.WithField("test", t.Name),
512:			log:    logging.WithField("test", t.Name),
528:			log:    logging.WithField("test", t.Name),
544:			log:    logging.WithField("test", t.Name),
560:			log:    logging.WithField("test", t.Name),
580:			log:    logging.WithField("test", t.Name),

controller/api/destination/watcher/ip_watcher_test.go
430:			endpoints := NewEndpointsWatcher(k8sAPI, logging.WithField("test", t.Name))
431:			watcher := NewIPWatcher(k8sAPI, endpoints, logging.WithField("test", t.Name))

controller/api/destination/watcher/profile_watcher_test.go
75:			watcher := NewProfileWatcher(k8sAPI, logging.WithField("test", t.Name))

controller/api/destination/endpoint_translator_test.go
105:		logging.WithField("test", t.Name),

controller/api/destination/server_test.go
100:	log := logging.WithField("test", t.Name)

controller/api/destination/watcher/endpoints_watcher_test.go
471:			watcher := NewEndpointsWatcher(k8sAPI, logging.WithField("test", t.Name))

controller/api/destination/watcher/traffic_split_watcher_test.go
88:			watcher := NewTrafficSplitWatcher(k8sAPI, logging.WithField("test", t.Name))
```

We should probably fix all of these (in a separate PR)

Reply:
Good catch, will make a note :+1: